README.md

```diff
@@ -1,5 +1,99 @@
-# q2-chemistree
-
-[![Build Status](https://travis-ci.org/biocore/q2-chemistree.svg?branch=master)](https://travis-ci.org/biocore/q2-chemistree) [![Coverage Status](https://coveralls.io/repos/github/biocore/q2-chemistree/badge.svg?branch=master)](https://coveralls.io/github/biocore/q2-chemistree?branch=master)
-
-Hierarchical orderings for mass spectrometry data
+# q2-chemistree
+
+A tool to build a tree of MS1 features for use with phylogenetic methods in metabolomics datasets.
+
+## Installation
+
+Once QIIME2 is [installed](https://docs.qiime2.org/2018.2/install/), activate your QIIME2 environment and install q2-chemistree following the steps below:
+
+```bash
+git clone https://github.com/biocore/q2-chemistree.git
+cd q2-chemistree
+pip install .
+qiime dev refresh-cache
+```
+
+We also need to download SIRIUS, a software-framework for de-novo identification of metabolites. q2-chemistree uses molecular substrucures predicted by this framework to build a hierarchy of the MS1 features in a dataset. SIRIUS is freely available [here](https://bio.informatik.uni-jena.de/software/sirius/).
+
+## Demonstration
+
+`q2-chemistree` ships with the following three methods:
+
+```
+qiime chemistree fingerprint
+qiime chemistree make-hierarchy
+qiime match-table
+```
+
+To generate a tree relating the MS1 features in your experiment, we need to pre-process mass-spectrometry data (.mzXML files) using [MZmine2](http://mzmine.github.io/) and produce the following:
+
+Inputs:
+
+1. An MGF file with both MS1 and MS2 information. This file will be imported into QIIME2 as a `MassSpectrometryFeatures` type.
+2. A feature table with peak areas of MS1 ions per sample. This table will need to be imported from a CSV file into the [BIOM](http://biom-format.org/documentation/biom_conversion.html) format, and then into QIIME as a `FeatureTable[Frequency]` artifact.
+
+These input files can be obtained following peak detection in MZmine2. [Here](https://github.com/anupriyatripathi/q2-chemistree/demo/batchQE-MZmine-2.33.xml) is an example MZmine2 batch file used to generate these.
+
+You can create a separate folder to store files for this demonstration with:
+
+```bash
+mkdir demo-chemistree
+cd demo-chemistree
+```
+
+Download a small feature table and MGF file using:
+
+```bash
+wget https://github.com/biocore/q2-chemistree/demo/feature-table.biom
+wget https://github.com/biocore/q2-chemistree/demo/sirius.mgf
+```
+
+Once you [import](https://docs.qiime2.org/2018.8/tutorials/importing/) these files into the appropriate QIIME2 artifact format, you can follow these steps to generate your tree. Alternatively, you can proceed by directly downloading the corresponding artifact files as follows:

```

Comment:
I suggest that we only include the BIOM and the MGF files, and that we remove the qza files from the repository. Also include commands to import the data:

```bash
qiime tools import --type MassSpectrometryFeatures --....
qiime tools import --type FeatureTable[Frequency] ...
```